var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -915,7 +914,9 @@ def __init__(self, spec, **kwargs):
         # Build WM_OPTIONS
         # ----
         # WM_LABEL_OPTION=Int$WM_LABEL_SIZE
-        # WM_OPTIONS=$WM_ARCH$WM_COMPILER$WM_PRECISION_OPTION$WM_LABEL_OPTION$WM_COMPILE_OPTION
+        # WM_OPTIONS=$WM_ARCH \
```

Comment:
This is misleading, there are no spaces in the resulting variable

Reply:
fixed.